ui/admin/app/components/form/credential/json/index.hbs

```diff
@@ -0,0 +1,116 @@
+<Rose::Form
+  @onSubmit={{@submit}}
+  @cancel={{@cancel}}
+  @disabled={{@model.isSaving}}
+  @showEditToggle={{if @model.isNew false true}}
+  as |form|
+>
+
+  <form.input
+    @name='name'
+    @value={{@model.name}}
+    @label={{t 'form.name.label'}}
+    @error={{@model.errors.name}}
+    @helperText={{t 'form.name.help'}}
+    disabled={{form.disabled}}
```

Comment:
I don't think both the disabled/readonly do anything here. `@disabled={{@model.isSaving}}` should take care of disabling the input fields when the form is disabled.

Reply:
Removed those attributes.